Subject: Fix i8042 PnP detection if only KBD port is detected
Date: Sun Mar 13 22:15:56 CET 2005
From: Vojtech Pavlik <vojtech@suse.cz>
===================================================================

ChangeSet@1.1994.7.4, 2005-03-09 07:50:58+01:00, vojtech@suse.cz
  input: Add a missing KERN_INFO message designation, fix behavior
         when only a keyboard part of the controller is detected.
  
  Signed-off-by: Vojtech Pavlik <vojtech@suse.cz>

 i8042-x86ia64io.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -Nru a/drivers/input/serio/i8042-x86ia64io.h b/drivers/input/serio/i8042-x86ia64io.h
--- a/drivers/input/serio/i8042-x86ia64io.h	2005-03-13 22:16:00 +01:00
+++ b/drivers/input/serio/i8042-x86ia64io.h	2005-03-13 22:16:00 +01:00
@@ -227,7 +227,7 @@
 	int result_kbd, result_aux;
 
 	if (i8042_nopnp) {
-		printk("i8042: PNP detection disabled\n");
+		printk(KERN_INFO "i8042: PNP detection disabled\n");
 		return 0;
 	}
 
@@ -265,7 +265,7 @@
 		i8042_pnp_kbd_irq = i8042_kbd_irq;
 	}
 
-	if (result_aux > 0 && !i8042_pnp_aux_irq) {
+	if (!i8042_pnp_aux_irq) {
 		printk(KERN_WARNING "PNP: PS/2 controller doesn't have AUX irq; using default %#x\n", i8042_aux_irq);
 		i8042_pnp_aux_irq = i8042_aux_irq;
 	}
